reporter/grpc.go

```diff
@@ -232,6 +236,9 @@ func (r *gRPCReporter) initSendPipeline() {
 				continue StreamLoop
 			}
 			for s := range r.sendCh {
+				if !r.rs(s) {
+					continue
+				} 
```

Comment:
```suggestion
				if r.rs != nil && !r.rs(s) {
					continue
				} 
```

I recommend changing it to this, for two reasons:
1. No need to set defaul `ReportStrategy`.
2. Avoid panic caused by user setting `nil`.

Reply:
Has been updated